Extensions/GoogleOptimize/Src/Tasks/GoogleOptimize/operations/TaskOperation.ts

```diff
@@ -0,0 +1,25 @@
+import {IVariation,IExperiment} from './../models/IOptimize';
+import {Optimizeclient} from './optimizeclient'
+import * as tl from 'azure-pipelines-task-lib/task';
+import * as schema from './../models/Schema.json'
+const fs = require('fs');
+
+export class TaskOperation {
+
+	private oxclient: Optimizeclient;
+
+	constructor() {
+		this.oxclient = new Optimizeclient();
+	}
+
+    public async stopExperiment(experiment: IExperiment): Promise < void > {
+		experiment.status = "ENDED";
```

Comment:
Use string enum for statuses (Ended, Running). So that if string changes there is only one touch point.